app/src/main/res/layout/activity_main_menu.xml

```diff
@@ -45,31 +68,78 @@
     </LinearLayout>
 
 
-    <LinearLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_weight="1"
         android:orientation="horizontal">
 
         <Button
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+            android:id="@+id/button"
+            android:layout_width="100dp"
+            android:layout_height="100dp"
+            android:layout_marginStart="4dp"
             android:layout_weight="1"
-            android:layout_margin="6dp"
-            android:text="Add Expense"
+            android:background="@drawable/button_add_expense"
+            android:foreground="@drawable/baseline_attach_money_24"
             android:onClick="onClickExpenseAdd"
-            />
+            android:text=""
+
+
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toStartOf="@+id/button2"
+            app:layout_constraintHorizontal_bias="0.245"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.019" />
 
         <Button
```

Comment:
Somewhere along this section, the statistics button is off-center to the right.

You could use a different layout for better flexibility on this section. It's a minor UX issue, but at least keep it in mind.